crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -176,29 +184,17 @@ impl core::ops::Deref for ClampedPercentage {
     }
 }
 
-#[derive(Debug, Clone)]
-pub struct RecordedBlock {
-    pub height: u32,
-    pub block_bytes: u64,
-    pub block_cost: u64,
-}
-
-#[derive(serde::Serialize, serde::Deserialize, Debug, Clone, PartialEq)]
-pub struct BlockBytes {
-    pub height: u32,
-    pub block_bytes: u64,
-}
-
 impl AlgorithmUpdaterV1 {
     pub fn update_da_record_data(
         &mut self,
-        blocks: &[RecordedBlock],
+        height_range: Range<u32>,
+        range_cost: u128,
     ) -> Result<(), Error> {
-        for block in blocks {
-            self.da_block_update(block.height, block.block_bytes, block.block_cost)?;
+        if !height_range.is_empty() {
+            self.da_block_update(height_range, range_cost)?;
+            self.recalculate_projected_cost();
+            // self.normalize_rewards_and_costs();
```

Comment:
Is this commented-out on purpose?

Reply:
Yes. I added the TODO with the related issue.